### DISCUSSION TITLE
Feature Request: Brightness Setting

### DISCUSSION THREAD

**Discussion by odudex** [selfcustody/krux#264] [Ideas]
By [HugoEnder](https://github.com/hugoender)

Add brightness setting to settings page. The default brightness of 1 (minimum) is a bit too dim when using the device in well lit areas. I understand that making the display brighter may make the QR codes harder to read for some cameras so my recommendation for that is to lower the brightness when the device displays a QR code.

Original: https://github.com/odudex/krux/issues/6

Additional info: Currently QR codes can toggle brightness on all devices when PAGE is pressed, and this works better with recent IRQ interface. But M5stickV(his device) still don't have a LCD backlight  setting

**Comment by tadeubas** [comment]
I would love to see this brightness setting on Maix Dock too!
